p2p/src/main/java/bisq/network/p2p/storage/payload/ProtectedMailboxStorageEntry.java

```diff
@@ -107,7 +107,7 @@ public boolean isValidForAddOperation() {
         if (!result) {
             String res1 = this.toString();
             String res2 = "null";
-            if (mailboxStoragePayload != null && mailboxStoragePayload.getOwnerPubKey() != null)
+            if (mailboxStoragePayload.getOwnerPubKey() != null)
```

Comment:
(applies to all removed null-checks)
you say in the commit comment that `fromProto` will throw an exception. I cannot see that. However, it seems like the protobuf implementation does supply a default value which is not null. So either way this change should be fine. Or did I miss something here?

Reply:
I could have been more clear with the distinction. Here is the relevant code for the payload itself. Since it is a submessage in proto3 it can technically be null, but the resolver handles those cases and raises an exception:

https://github.com/bisq-network/bisq/blob/76e8c5736a6c5f56cca83323c373d2f3eb3f2542/core/src/main/java/bisq/core/proto/network/CoreNetworkProtoResolver.java#L249-L278

The other non-null issues were due to the initialized value for proto3 not being null, this isn't one of those cases.

Reply:
ah damn. missed that.